File: FMI/ANUL II/SEM 2/MPP/ProiectJava/Zboruri/ZboruriNetworking/src/main/java/ro/mpp2024/network/jsonprotocol/Response.java
package ro.mpp2024.network.jsonprotocol;

import ro.mpp2024.domain.Zbor;

import java.io.Serializable;
import java.util.List;


public class Response implements Serializable {
    private ResponseType Type;
    private String ErrorMessage;
    private List<Zbor> Flights;
    private Zbor Zbor;
    private Zbor OldZbor;

    public Response() {
    }

    public ResponseType getType() {
        return Type;
    }

    public void setType(ResponseType type) {
        this.Type = type;
    }

    public String getErrorMessage() {
        return ErrorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        this.ErrorMessage = errorMessage;
    }

    public List<Zbor> getFlights() {
        return Flights;
    }

    public void setFlights(List<Zbor> flights) {
        this.Flights = flights;
    }

    public void setZbor(Zbor zbor) {
        this.Zbor = zbor;
    }

    public Zbor getZbor() {
        return Zbor;
    }

    public void setOldZbor(Zbor oldZbor) {
        this.OldZbor = oldZbor;
    }

    public Zbor getOldZbor() {
        return OldZbor;
    }
}

File: FMI/ANUL II/SEM 2/MPP/ProiectCSharp/Zboruri/Networking/Response.cs
using Model;

namespace Networking;

public class Response
{
    public ResponseType Type { get; set; }
    public String ErrorMessage { get; set; }
    public List<Zbor> Flights { get; set; }
    public Zbor Zbor { get; set; }
    public Zbor OldZbor { get; set; }
    public Zbor Found { get; set; }

    public override string ToString()
    {
        return
            $"{nameof(Type)}: {Type}, {nameof(ErrorMessage)}: {ErrorMessage}, {nameof(Flights)}: {Flights}, {nameof(Zbor)}: {Zbor}, {nameof(OldZbor)}: {OldZbor}";
    }
}